### ISSUE TITLE
TextBuffer.on is deprecated.

### ISSUE THREAD

**Issue by smlu** [leny/atom-w3c-validation#7]
Use TextBuffer::onDidSave instead. A TextBuffer instance is no longer provided as a callback argument.

```
TextBuffer.on (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1182:16)
<unknown> (/Users//.atom/packages/w3c-validation/lib/main.coffee:50:66)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:272:16)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:329:18)
Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
PaneContainer.addedPaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:375:27)
```

**Comment by leny** [comment]
Fixed in 0.2.0. Thanks for the report.

**Comment by smlu** [comment]
Seems to be working properly now. Thx
